src/wizard/ionic/index.md

```diff
@@ -0,0 +1,217 @@
+---
+name: Capacitor
+doc_link: https://docs.sentry.io/platforms/javascript/guides/capacitor/
+support_level: production
+type: framework
+---
+
+Install the Sentry Capacitor SDK alongside the sibling Sentry SDK:
+
+```angular {tabTitle: Ionic + Angular}
+# npm
+npm install --save @sentry/capacitor @sentry/angular
+
+# yarn
+yarn add @sentry/capacitor @sentry/angular @sentry/tracing
```

Comment:
We have tracing here but not on the npm example. Which one is correct?

Reply:
by including @sentry/capacitor it also includes @sentry/tracing so I'll just remove it form yarn